src/TLuaInterpreter.cpp

```diff
@@ -52,6 +52,22 @@
 #include <QRegularExpression>
 #include "post_guard.h"
 
+#include <QRegularExpression>
+#include <QSound>
+#include <QSslConfiguration>
+#include <QString>
+#include <QStringList>
+#include <QStringBuilder>
+
+#include <assert.h>
+#include <list>
+#include <string>
+
+//#undef lua_register
+//#define lua_register(L,n,f) ( mLuaFunctionNames << n, lua_pushcfunction(L, (f)), lua_setglobal(L, (n)))
+
+
+#include "post_guard.h"
```

Comment:
These changes look like a merge gone bad, could you have a look?

Reply:
I can take a look. There are tons of unnecessary comments sitting around in there.